core/google/cloud/client.py

```diff
@@ -64,7 +64,7 @@ def from_service_account_json(cls, json_credentials_path, *args, **kwargs):
 
         :rtype: :class:`_ClientFactoryMixin`
         :returns: The client created with the retrieved JSON credentials.
-        :raises: :class:`TypeError` if there is a conflict with the kwargs
+        :raises TypeError: if there is a conflict with the kwargs
                  and the credentials created by the factory.
```

Comment:
This indent used to be aligned with the `:class:...`. Your call on how to "fix" / if it needs to be "fixed"